api/app/views/mno_enterprise/jpi/v1/app_instances/_resource.json.jbuilder

```diff
@@ -8,6 +8,15 @@ json.oauth_keys_valid app_instance.oauth_keys_valid
 #json.microsoft_trial_url app_instance.microsoft_trial_url
 json.created_at app_instance.created_at
 
+if app_instance.per_user_licence?
+	json.per_user_licence app_instance.per_user_licence
+	json.licences_count app_instance.active_licences_count
+end
+
+if app_instance.under_free_trial?
+  json.free_trial_end_at time_ago_in_words(app_instance.free_trial_end_at)
```

Comment:
**IN MNO-ENTERPRISE-ANGULAR**
Hey @alexnoox, once we have in the controller the result of moment(), lets say:
```
$scope.app.free_trial_end_at = moment(app.free_trial_end_at).format('dddd, MMMM Do YYYY, h:mm:ss a')
```
and in the view:
```
{{ 'mno_enterprise.templates.impac.dock.settings.free_trial' | translate }}: {{ app.free_trial_end_at }}
```
And displayed:
```
Free Trial: Thursday, April 6th 2017, 12:09:25 pm
```
how would you suggest to integrate i18n?

-I was thinking about using as result from moment something like 31-01-2017, and use the i18n the same way we did it with the pluralisation. (seems like a dirty approach)

-As other solution,  moment() seems to handle i18n:
https://momentjs.com/docs/#/i18n/

May you please hint me on this?
Thanks a lot.